src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -987,6 +987,40 @@ public class DeleteProjectById : IReturnVoid
         public string Id { get; set; }
     }
 
+    [Route("/v1/resultgrades", "GET")]
+    public class GetResultGrades : IReturn<SearchResultResultGrade>
+    {
+        
+    }
+
+    [Route("/v1/resultgrades", "POST")]
+    public class PostResultGrade : IReturn<ResultGrade>
+    {
+        public string Id { get; set; }
+        public string CustomId { get; set; }
+        public AuditAttributes AuditAttributes { get; set; }
+    }
+
+    [Route("/v1/resultgrades/{id}", "GET")]
+    public class GetResultGrade : IReturn<ResultGrade>
+    {
+        public string Id { get; set; }
+    }
+
+    [Route("/v1/resultgrades/{id}", "PUT")]
+    public class PutSparseResultGrade : IReturn<ResultGrade>
+    {
+        public string Id { get; set; }
+        public string CustomId { get; set; }
+        public AuditAttributes AuditAttributes { get; set; }
+    }
+
+    [Route("/v1/resultgrades/{id}", "DELETE")]
+    public class DeleteResultGradeById : IReturnVoid
```

Comment:
The configurable result grades aren't yet in use. I think it would make sense to exclude these for now and introduce them in the next release.

Reply:
That's not really how the client code generator works. It generates the code from the Swagger spec your app defines.

Is this the only thing to omit? (I can manually tweak the generated code by hand, but that isn't something I want to keep having to manually remember each time).

If calling this API just returns a 4xx or 5xx response right now without harming the state of the data on the server, we could also just leave it in.

How would you like to proceed?

Reply:
Yes, we should have done that on our side and excluded this from the documentation. This API should work fine, but the data that is created cannot be used anywhere, so it might be confusing. It's not a big problem if it stays, but I think it would be preferable if you manually excluded it. I agree that in general the tool should just use our API documentation.

Reply:
So is it just the one `DELETE /v1/resultgrades/{Id}` operation that should be removed? Or all the `/v1/resultgrades` routes?

What should I remove?

Reply:
All the resultgrades routes

Reply:
OK, I pushed that change.